test/libsolidity/syntaxTests/inheritance/allow_empty_duplicated_super_constructor_call.sol

```diff
@@ -0,0 +1,2 @@
+contract A { constructor() public { } }
+contract B is A { constructor() A() public {  } }
```

Comment:
Is there a test that just has `A` without parentheses? This is supported for modifiers, but perhaps we should disallow it later for base constructors.